docs/NooBaaNonContainerized/Configuration.md

```diff
@@ -116,13 +118,31 @@ certificates/                                   // Optional
 `accounts/` - 
 * <u>Type</u>: Directory.
 * <u>Required</u>: Yes.
-* <u>Description</u>: A directory that contains configuration files for individual accounts, each account configuration file is named {account_name}.json and adheres to the [account schema](../../src/server/system_services/schemas/nsfs_account_schema.js).
+* <u>Description</u>: A directory that contains configuration files for individual accounts, each account configuration file is named {account_id}.json and adheres to the [account schema](../../src/server/system_services/schemas/nsfs_account_schema.js).
 * <u>Example</u>:
     ```sh
     > ls /etc/noobaa.conf.d/accounts/
-    alice.json
-    bob.json
-    charlie.json
+    abcd.json
+    1234.json
+    ab12.json
+    ```
+`root_accounts/
+* <u>Type</u>: Directory.
+* <u>Required</u>: Yes.
+* <u>Description</u>: A directory that contains a directory per root account, named {root_account_name}.
+Inside each such direcoty, there is a symlink per iam account that belong to the root account, named {account_name}.symlink.
+There is also a symlink for the root account, named  {root_account_name}.symlink.
+Symlinks link to an account withing account/.
```

Comment:
general note - I noticed a few typos in the docs, try to check with automated tool the newly added text, for example:
```suggestion
Inside each such directory, there is a symlink per iam account that belongs to the root account, named {account_name}.symlink.
There is also a symlink for the root account, named  {root_account_name}.symlink.
Symlinks link to an account within `account/`.
```